docs/source/getting-started/typical-workflow.rst

```diff
@@ -5,15 +5,8 @@ A differentially private analysis in OpenDP typically has the following steps:
 
 1. Identify the unit of privacy
 2. Set privacy loss parameters
-3. Collect public information
-4. Mediate access to data
-5. Submit DP queries
-
-.. Diagram source: https://docs.google.com/drawings/d/1W4l9x3UM3hbVLWlC0nzijqgaQ31wY5ERebp8jkYy1yc/edit
-
-.. image:: code/typical-workflow-diagram.svg
-    :width: 60%
-    :alt: Diagram representing typical data flow with OpenDP, from a raw CSV to a differentially private release. 
+3. Mediate access to data
+4. Submit DP queries (examples: count and sum)
```

Comment:
At this point in the docs, I think we would do well to keep it simple. What is the benefit of doing count and sum? Could it be left to a later section?